### PR TITLE
Swope api

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: applications/gitops_config.adoc [hunk 1]
```diff
@@ -64,4 +64,4 @@ spec:
     name: all-openshift-clusters
```
Comment (haoqing0110) [inline applications/gitops_config.adoc:64]: v1beta1

File: clusters/placement_managed.adoc [hunk 2]
```diff
@@ -145,7 +145,7 @@ spec:
 +
 [source,yaml]
 ----
-apiVersion: cluster.open-cluster-management.io/v1alpha1
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc:148]:
``` 
     - name: ResourceAllocatableMemory 
```
-->
```
      - scoreCoordinate:
          builtIn: ResourceAllocatableMemory
```

File: clusters/placement_managed.adoc [hunk 3]
```diff
@@ -161,7 +161,7 @@ spec:
...
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc:164]:
```
  prioritizerPolicy:
    configurations:
      - name: ResourceAllocatableCPU
        weight: 2
      - name: ResourceAllocatableMemory
        weight: 2
```
--->
```
  prioritizerPolicy:
    configurations:
      - scoreCoordinate:
          builtIn: ResourceAllocatableCPU
        weight: 2
      - scoreCoordinate:
          builtIn: ResourceAllocatableMemory
        weight: 2
```

File: clusters/placement_managed.adoc [hunk 4]
```diff
@@ -176,11 +176,11 @@ spec:
         weight: 2
 ----
 
-- Select a cluster with the largest allocatable memory and pin the placementdecisions. 
+- Select two clusters with the largest allocatable memory and the largest add-on score cpu ratio, and pin the placement decisions. 
 +
 [source,yaml]
 ----
-apiVersion: cluster.open-cluster-management.io/v1alpha1
```
Comment (haoqing0110) [inline clusters/placement_managed.adoc:183]:
```
apiVersion: cluster.open-cluster-management.io/v1beta1
kind: Placement
metadata:
  name: placement8
  namespace: ns1
spec:
  numberOfClusters: 2
  prioritizerPolicy:
    mode: Exact
    configurations:
      - scoreCoordinate:
          builtIn: ResourceAllocatableMemory
      - scoreCoordinate:
          builtIn: Steady
        weight: 3
      - scoreCoordinate:
          type: AddOn
          addOn:
            resourceName: default
            scoreName: cpuratio
```

**Review by haoqing0110 — COMMENTED** [review]
A few examples need update, see comments.

**Comment by swopebe** [comment]
Look and see if these changes are OK

**Review by haoqing0110 — APPROVED** [review]

**Pull request merged**